src/config-schema.ts

```diff
@@ -54,7 +54,7 @@ export const configSchema = {
     ).join(', ')}`,
     _elements: {
       _type: Type.String,
-      _validators: [validators.oneOf(Object.values(BillingCondition)), validateConditionGroup],
+      _validators: [validateConditionGroup],
```

Comment:
In the validateConditionGroup, we can add a check if the added conditions are part of the `BillingCondition` or not.
Thanks!